blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -100,3 +100,23 @@ Example::
    +----------+----------------------------------------+
    SELECT 4 rows in set (... sec)
 
+
+Native Window Functions
```

Comment:
I think we shouldn't use "native" as the concepts of native and "non-native" don't really exist afaik. Aggregates are functions that are optionally treated and executed as window functions (so from an execution perspective they are "native" window functions as well)

Reply:
ok, changing this to "general-purpose" wf